documentation/markdown_and_code_dev.md

```diff
@@ -9,44 +9,67 @@ These include {term}`Jupyter Notebook`s, Linux {term}`command-line <terminal>` a
 To read more about the differences between `.md` and `.ipynb`, see the [MyST Markdown documentation](https://mystmd.org/guide/md-vs-ipynb).
 
 ### JupyterLab Markdown Dependencies
 
 On the Fornax Science Console, all required dependencies for using Markdown in JupyterLab are pre-installed.
+
 To use Markdown notebooks in JupyterLab outside of the Fornax Science Console, ensure the following tools are installed:
 
 -   [`jupytext`](https://github.com/mwouts/jupytext) Python library
 -   [`jupyterlab-myst`](https://github.com/executablebooks/jupyterlab-myst) JupyterLab extension
 
+(open-md-in-jupyterlab)=
 ### Opening `.md` Files in JupyterLab
 
-1.  Navigate to the `.md` file in the JupyterLab file browser.
-2.  Right-click the file, select **Open With**, and choose **Notebook** or **Jupytext Notebook**.
-3.  The Markdown file will open in an interactive notebook interface with cells and execution capabilities—just like a traditional `.ipynb` notebook.
+In the Fornax Science Console, you can simply double click on a `.md` file in the file browser.
+It will open in an interactive notebook interface with cells and execution capabilities — just like a traditional `.ipynb` notebook.
+If you need to save the notebook output, pair it with a `.ipynb` document before running it by selecting: `File → Jupytext → Pair Notebook with ipynb document`.
+
+If you are not in the Fornax Science Console then the default viewer for `.md` files might be different.
+In that case, right-click the file, select **Open With**, and choose **Notebook** or **Jupytext Notebook** to open it in the interactive notebook interface.
 
-### Converting `.md` Markdown Notebooks to `.py` Scripts
+### Converting To and From `.md` Format
 
-It is easy to convert from `.md` Markdown to a `.py` script, if that fits better into your workflow:
+:::{tip}
+It is not necessary to convert `.md` to `.ipynb` before using a notebook!
+Notebooks in `.md` format can be used in JupyterLab just like a traditional notebook.
+See {ref}`open-md-in-jupyterlab`.
+:::
+
+It is easy to convert notebooks between file formats like `.md`, `.ipynb`, and `.py` using [jupytext](https://jupytext.readthedocs.io/):
 
 1.  Open a terminal.
 2.  Install **jupytext** if necessary (it is pre-installed in the Fornax Science Console):
 
     ```pip install jupytext```
 
-3.  Navigate to the directory containing your `.md` notebook file.
-4.  Run the following command to convert it to a Python script:
+3.  Navigate to the directory containing your notebook file.
+4.  Run a `jupytext` command to convert the format.
+    Some examples are:
+    -   Convert `.ipynb` to a MyST Markdown `.md` notebook:
+
+        ```jupytext --from notebook --to myst <your_notebook_file>.ipynb```
 
-    ```jupytext --to script <your_notebook_file>.md```
+    -   Convert `.md` to a `.ipynb` notebook (*not necessary for use in JupyterLab*):
+
+        ```jupytext --from notebook --to myst <your_notebook_file>.ipynb```
```

Comment:
I'm anticipating that you might not want this here so I wanted to point it out and mention my reasoning. I added it because people have asked how to do it more than once. One example is https://github.com/nasa-fornax/fornax-demo-notebooks/issues/372. If we leave this in here then we can update the fornax-demo-notebooks readme to just point to this page and then we can close that issue. But if you feel strongly that we should take it out, that's fine with me.

Reply:
No, you don't need this, or if you do, then do fix it up as it is a copy paste error atm
```suggestion
    -   Convert `.md` to a `.ipynb` notebook (*not necessary for use in JupyterLab*):

        ```jupytext --to ipynb <your_notebook_file>.md```
```